### PR TITLE
Rss feed

### PR DESCRIPTION
**Pull request by borntofrappe**
- I added `site.json` to provide general information on the website. I hope I'm not presumptuous to take the fields from Hylia's feed

- I looked the Atom format and picked `<summary>` and `<category>` elements to describe the summary and each tag for the items

Hopefully, it's worth something 🤞

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ module.exports = config => {
     return collection.getAll().find(x => x.url === '/');
   });
 
+  config.addCollection('items', function(collection) {
+    return collection.getFilteredByGlob('./src/item/*.md');
```
Comment (hankchizljaw) [inline .eleventy.js]: This is spot on! Maybe one thing that would be cool would be ordering by date, reversed?

File: src/feed.njk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <title>{{ item.data.title }}</title>
+    <link href="{{ item.data.link }}"/>
+    <id>{{ item.data.link }}</id>
+    {% for tag in item.data.tags %}
+      <category>{{ tag }}</category>
+    {% endfor %}
+    <summary type="html"><![CDATA[
+      {{ item.data.summary }}
+    ]]></summary>
+  </entry>
```
Comment (hankchizljaw) [inline src/feed.njk]: Shall we just indent this in so it's tidy?

File: .eleventy.js (hunk 1 above)
Comment (borntofrappe) [reply .eleventy.js]: It makes a lot of sense, so that the feed provides the latest entries. I'll try to find a way 👍

File: src/feed.njk (hunk 2 above)
Comment (borntofrappe) [reply src/feed.njk]: Could have swore I had intendet the code better, I'll fix it immediately.

**Comment by borntofrappe** [comment]
I hope I addressed both issues with this commit. I thought sorting the items would be more challenging, but eleventy's documentation helped a lot.

**Comment by hankchizljaw** [comment]
Yay! This is all amazing. Thank you so much! Merging :)

**Pull request merged**